tests/testthat/test-variable-transforms.R

```diff
@@ -49,12 +49,12 @@ insrts_list <- list(list(anchor = 0, name = "First one",
 insrts <- Insertions(data=insrts_list)
 
 test_that("findInsertPosition", {
```

Comment:
It might be worth adding a couple of tests with some invalid `last_category` values like `NULL`. This is just to be defensive against future changes.

Reply:
I've added a check on the one calling function. Since `findInsertPosition` is intended to be pretty internal I don't think we need to have input checks that passing a `NULL` there works in most cases except for the one that it absolutely couldn't work in. I'm inclined to leave the test for catching the problem with `collateCats` and leave it be. Unless we need to expand the use of `findInsertPosition` elsewhere / export it.